package.json

```diff
@@ -595,7 +633,80 @@
         "name": "typescript-deno-plugin",
         "enableForWorkspaceTypeScriptVersions": true
       }
-    ]
+    ],
+    "views": {
+      "explorer": [
+        {
+          "id": "denoTasks",
+          "name": "Deno Tasks",
+          "icon": "deno-icon.png",
+          "contextualTitle": "Deno Tasks",
+          "visibility": "hidden",
+          "when": "deno:lspReady",
+          "type": "tree"
+        }
+      ]
+    },
+    "menus": {
+      "commandPalette": [
+        {
+          "command": "deno.client.runTask",
+          "when": "false"
+        },
+        {
+          "command": "deno.client.debugTask",
+          "when": "false"
+        },
+        {
+          "command": "deno.client.openTaskDefinition",
+          "when": "false"
+        },
+        {
+          "command": "deno.client.refreshTasks",
+          "when": "false"
+        }
+      ],
```

Comment:
@ArmaanAS Does this section do anything considering all the conditions are `false`? They don't show up on the command palette, though `deno.client.refreshTasks` is the only one that should.

Reply:
Without these lines added, all of these commands will show up in the command palette but they won’t function properly.

Maybe refreshTasks could be displayed in the command palette, if you think that’s useful.